lib/analyzers/bower/index.spec.js

```diff
@@ -0,0 +1,49 @@
+import test from 'ava';
+import tmp from 'tmp';
+import fs from 'fs';
+import getCredits from './';
+
+test( 'getCredits - get availalbe bower credits', t => {
+  tmp.dir( { unsafeCleanup : true }, ( error, path, cleanUpCb ) => {
+    fs.mkdirSync( `${path}/bower_components` );
+
+    fs.mkdirSync( `${path}/bower_components/foo` );
+    fs.writeFileSync(
+      `${path}/bower_components/foo/bower.json`,
+      JSON.stringify( {
+        authors : [
+          'Bob Calsow'
+        ]
+      } ),
+      'utf8'
+    );
+
+
+    fs.mkdirSync( `${path}/bower_components/bar` );
+    fs.writeFileSync(
+      `${path}/bower_components/bar/bower.json`,
+      JSON.stringify( {
+        authors : [
+          {
+            name  : 'Alice Bobson',
+            email : 'alicebobson@@alison.io'
```

Comment:
@@ in the mail Adresse. ;)

Radimir Bitsov <notifications@github.com> schrieb am Fr., 1. Jan. 2016,
11:47:

> In lib/analyzers/bower/index.spec.js
> <https://github.com/stefanjudis/credits/pull/10#discussion_r48677154>:
>
> > +        authors : [
> > +          'Bob Calsow'
> > +        ]
> > +      } ),
> > +      'utf8'
> > +    );
> > +
> > +
> > +    fs.mkdirSync( `${path}/bower_components/bar` );
> > +    fs.writeFileSync(
> > +      `${path}/bower_components/bar/bower.json`,
> > +      JSON.stringify( {
> > +        authors : [
> > +          {
> > +            name  : 'Alice Bobson',
> > +            email : 'alicebobson@@alison.io'
>
> can you please point me to the exact line with the typo?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/stefanjudis/credits/pull/10/files#r48677154>.
>